recipes-azure-iot/iot-hub-device-update/iot-hub-device-update/swupdate_handler_v2_grub.sh

```diff
@@ -437,6 +437,12 @@ while [[ $1 != "" ]]; do
         $ret 0
         ;;
 
+    --*)
+	error "unrecognized option \"$1\""
+        print_help
+        $ret 1
+	;;
+
```

Comment:
what does it have to do with health check and what was the reason you introduced this change?

Reply:
yeah, right, it has nothing to do with health check, I just stumbled over that you can give whatever options without the script complaining what I found embarrassing.
I can take that out here, but I find that bad behavior nevertheless which should probably targeted independently from this PR.

Reply:
removed.